### PR TITLE
Replace lodash with lodash subpackages

### PR DESCRIPTION
**Pull request by JackuB**
This PR limits the usage of `lodash` package, in favor of `lodash.<module>` packages.

Also adds an eslint rule to prevent lodash being used again.

### CODE REVIEW

File: test/acceptance/cli-test/iac/cli-test.iac-utils.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as _ from 'lodash';
+const sortBy = require('lodash.sortby');
```
Comment (aron) [inline test/acceptance/cli-test/iac/cli-test.iac-utils.ts:1]: Any reason to write it this way over `import * as sortBy from 'lodash.sortby'` ?

**Review by aron — APPROVED** [review]
LGTM for @snyk/cloudconfig files

**Review by gitphill — APPROVED** [review]

File: test/acceptance/cli-test/iac/cli-test.iac-utils.ts (hunk 1 above)
Comment (JackuB) [reply test/acceptance/cli-test/iac/cli-test.iac-utils.ts:1]: Screen Shot 2021-02-12 at 15 02 23

Sadly, you can't use it like this with our setup

File: package.json [hunk 2]
```diff
@@ -113,9 +129,25 @@
   "devDependencies": {
     "@types/agent-base": "^4.2.1",
     "@types/diff": "^3.5.2",
-    "@types/graphlib": "^2.1.7",
     "@types/jest": "^25.2.3",
     "@types/lodash": "^4.14.161",
+    "@types/lodash.assign": "^4.2.6",
```
Comment (maxjeffos) [inline package.json]: I don't think we need the `@types/lodash.xyz` for those lodash dot packages we are not importing.

**Review by maxjeffos — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (JackuB) [reply package.json]: Removed them

**Pull request merged**